Support/lib/build.rb

```diff
@@ -29,6 +29,26 @@ def cargo_params
   ENV.key?('TM_CARGO_PARAMS') ? ENV['TM_CARGO_PARAMS'].split(' ') : []
 end
 
+# Try to use cargo metadata to find the workspace
+# root for resolving relative paths in error messages.
+# Fall back to TM_PROJECT_DIRECTORY if that attempt
+# fails for any reason.
+def get_worspace_root(cargo_home)
```

Comment:
Hm, is this function called `get_worspace_root` rather than `get_workspace_root` for any particular reason?

Reply:
Nope, just a typo.